### ISSUE TITLE
Identify the purpose of fields programmatically

### ISSUE THREAD

**Issue by unclesam2024** [web-infrastructure/doc_architecture#51]
To comply with WCAG 2.1, fields must identify what their purpose is programmatically. If done correctly, this allows browsers to help users fill in forms with known information, such as their name and email address.

This is accomplished via the `autocomplete attribute`, which should be set to a [supported value](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) wherever appropriate, e.g.

`<label for="name">First Name</label>`
`<input id="name" type="text" autocomplete="given-name">`

For fields where autocomplete is not appropriate, e.g. due to security, or the field not matching the specification, the specific field should be ignored.

See [how to use HTML 5.2 attributes](https://www.w3.org/WAI/WCAG21/Techniques/html/H98) for more information.
